### DISCUSSION TITLE
Its a bird dog!

### DISCUSSION THREAD

**Discussion by stevedee** [mcguirepr89/BirdNET-Pi#1088] [Q&A] [answered]
Once in a while I get a detection classified as a raven. Checking the recordings, its always turned out to be a neighbours dog ..._so far._

I've changed the classification name to Raven<>Dog as a reminder to check the recording.
![Raven_Dog](https://github.com/mcguirepr89/BirdNET-Pi/assets/1870807/54b073a9-72d8-48ee-bcdd-b8a28dc6fee3)

I've also raised the confidence level from 70% to 75%, which has reduced the frequency of false detection.
But I was just wondering if anyone else has this problem with dogs.

**Comment by pddpauw** [comment, marked as answer]
have had the same, and especially on those lower frequencies those false positives can happen.
I even once had that the sound of a closing gate was recorded as an owl species.

this method has its limitations, which is why checking the wav files afterwards is useful.  Typically, if I have one observation of a relative rare species, I always check the recording and correct if needed

**Reply by stevedee** [reply]
Yes, its a dog with a deep voice.
A dog much closer with a higher pitched bark does not register as a bird.
